lib/github-pages-health-check/domain.rb

```diff
@@ -8,7 +8,8 @@ class Domain < Checkable
         207.97.227.245
         204.232.175.78
         199.27.73.133
-      ].freeze
+        199.27.76.133
```

Comment:
LGTM. Could we maybe add comments here? I have the worst time remembering what IP's are what.